TestResultsQualityGate/TestResultsQualityService.ts

```diff
@@ -91,7 +113,18 @@ export class TestResultsQualityService {
                 return;
             }
             // Get test results in current build
-            const currentTestResults = await this.apiClient.getTestResultsByBuildId(Number(this.buildId));
+            let currentTestResults = [];
+            if (this.pipelineType == PipelineTypeEnum.BUILD) {
+                // Build pipeline
+                currentTestResults = await this.apiClient.getTestResultsByBuildId(Number(this.buildId));
+            } else {
+                // Release pipeline
+                if(this.type == TypeEnum.NEWLY_FAILED_TESTS) {
+                    tl.setResult(tl.TaskResult.SucceededWithIssues, `Test results quality gate for 'Newly failed tests' type is not currently supported in the release pipeline, if you require this feature, please contact Parasoft support.`);
```

Comment:
Customer would contact us if they really need this feature. Please change it to:
The test results quality gate for 'Newly failed tests' is not currently supported in the release pipeline.